lib/taskana-core/src/main/java/pro/taskana/workbasket/internal/WorkbasketServiceImpl.java

```diff
@@ -1158,27 +1138,45 @@ private boolean skipAuthorizationCheck(WorkbasketPermission... requestedPermissi
   private void validateWorkbasket(Workbasket workbasket)
       throws InvalidWorkbasketException, DomainNotFoundException {
     // check that required properties (database not null) are set
+    validateNameAndType(workbasket);
```

Comment:
If we re-use the whole validateWorkbasket method, then we woudln't have to refactor this to a separate method, right?

Reply:
True, but see above